upload/admin/controller/sale/subscription.php

```diff
@@ -658,9 +658,8 @@ public function info(): void {
 
 		// Product data
 		if (!empty($subscription_info)) {
-			//$this->load->model('sale/order');
-			//getSubscriptionByOrderProductId
-		//	$product_info = $this->model_sale_order->getProductByOrderProductId($subscription_info['order_id'], $subscription_info['order_product_id']);
+			$this->load->model('sale/order');
+			$product_info = $this->model_sale_order->getProductByOrderProductId($subscription_info['order_id'], $subscription_info['order_product_id']);
```

Comment:
I would like to ask for your help, as this function is not implemented in the model. 
Error message:
Notice: Undefined property: Proxy::getProductByOrderProductId in /.../admin/controller/sale/subscription.php on line 671: in /.../public/system/engine/proxy.php on line 85

Reply:
Can we open an issue

Reply:
Yes, I will open an issue.
I also find multiple subscription related issue.